STYLE.md

```diff
@@ -0,0 +1,39 @@
+# API style guidelines
+
+Generally follow guidance at https://cloud.google.com/apis/design/, in
+particular for proto3 as described at
+https://cloud.google.com/apis/design/proto3. In addition, the following
+conventions should be followed:
+
+* The API is primarily intended for machine generation and consumption. It is
+  expected that the management server is responsible for mapping higher level
+  configuration concepts to API responses. Similarly, static configuration
+  fragments may be generated by templating tools, etc. The APIs and tools
+  used to generate xDS configuration are beyond the scope of the definitions in
+  this repository.
+
+* Use [wrapped scalar
+  types](https://github.com/google/protobuf/blob/master/src/google/protobuf/wrappers.proto)
+  where there is a need for a default value that does not match the proto3
```

Comment:
a need => a real need

Reply:
I can make some change to the language here, but I feel there are totally legitimate places to use wrappers and that this isn't an API design anti-pattern, so we shouldn't call it out as one. Take for example these places we use wrappers for non-proto3 defaults in the API today:

* https://github.com/envoyproxy/data-plane-api/blob/master/api/cds.proto#L95
* https://github.com/envoyproxy/data-plane-api/blob/master/api/cds.proto#L231
* https://github.com/envoyproxy/data-plane-api/blob/master/api/cds.proto#L378

(and that's just a fraction of them).